ops/demo/alerts.tf

```diff
@@ -3,8 +3,14 @@ module "metric_alerts" {
   env                 = local.env
   app_service_plan_id = module.simple_report_api.app_service_plan_id
   app_service_id      = module.simple_report_api.app_service_id
-  action_group_id     = data.terraform_remote_state.global.outputs.slack_alert_action_id
   severity            = 1
   rg_name             = data.azurerm_resource_group.rg.name
   tags                = local.management_tags
+  mem_threshold       = 85
+
+  action_group_ids = [
+    data.terraform_remote_state.global.outputs.slack_alert_action_id,
+    data.terraform_remote_state.global.outputs.pagerduty_demo_action_id
```

Comment:
What is this - separate PR change?

Reply:
Yes, the service code is in #1500. I think probably I should have done them both in one go, but I try to separate out the repetitive parts from the interesting parts.